packit_service/config.py

```diff
@@ -251,6 +251,7 @@ def get_service_config(cls) -> "ServiceConfig":
         return cls.service_config
 
     def get_project_to_sync(self, dg_repo_name, dg_branch) -> Optional[ProjectToSync]:
+        # TODO: Is it ok that we don't check namespace? Can't this be misused from a fork?
```

Comment:
in such a case, the build should fail - one should not be able to submit production builds from a fork

Reply:
But this is about upstream->downstream sync that I had to fix.

Reply:
ah, sorry, did not understand it then